### ISSUE TITLE
libmysqlclient fails to build on i686-w64-mingw32.static.posix

### ISSUE THREAD

**Issue by uwehermann** [mxe/mxe#1346]
```
Failed to build package libmysqlclient for target i686-w64-mingw32.static.posix!
------------------------------------------------------------
Makefile:149: recipe for target 'all' failed
make[2]: *** [all] Error 2
make[2]: Leaving directory '/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src.build'
Makefile:548: recipe for target 'build-only-libmysqlclient_i686-w64-mingw32.static.posix' failed
make[1]: *** [build-only-libmysqlclient_i686-w64-mingw32.static.posix] Error 2
make[1]: Leaving directory '/home/uwe/mxe-git'
real    1m30.592s
user    2m41.040s
sys     0m9.720s
------------------------------------------------------------
[log]      /home/uwe/mxe-git/log/libmysqlclient_i686-w64-mingw32.static.posix

Makefile:548: recipe for target '/home/uwe/mxe-git/usr/i686-w64-mingw32.static.posix/installed/libmysqlclient' failed
make: *** [/home/uwe/mxe-git/usr/i686-w64-mingw32.static.posix/installed/libmysqlclient] Error 1
```

Tail of log:
```
/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/extra/perror.c: In function 'print_win_error_msg':
/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/extra/perror.c:200:7: warning: format '%d' expects argument of type 'int', but argument 2 has type 'DWORD' [-Wformat=]
       printf("Win32 error code %d: %s", error, s);
       ^
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_exit':
/home/uwe/mxe-git/tmp-gcc-i686-w64-mingw32.static.posix/gcc-4.9.3/mingw-w64-v4.0.6/mingw-w64-libraries/winpthreads/src/thread.c:961: multiple definition of `pthread_exit'
../mysys/libmysys.a(my_winthread.c.obj):/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:185: first defined here
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_cancel':
/home/uwe/mxe-git/tmp-gcc-i686-w64-mingw32.static.posix/gcc-4.9.3/mingw-w64-v4.0.6/mingw-w64-libraries/winpthreads/src/thread.c:1078: multiple definition of `pthread_cancel'
../mysys/libmysys.a(my_winthread.c.obj):/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:247: first defined here
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_attr_init':
/home/uwe/mxe-git/tmp-gcc-i686-w64-mingw32.static.posix/gcc-4.9.3/mingw-w64-v4.0.6/mingw-w64-libraries/winpthreads/src/thread.c:1215: multiple definition of `pthread_attr_init'
../mysys/libmysys.a(my_winthread.c.obj):/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:25: first defined here
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_attr_destroy':
/home/uwe/mxe-git/tmp-gcc-i686-w64-mingw32.static.posix/gcc-4.9.3/mingw-w64-v4.0.6/mingw-w64-libraries/winpthreads/src/thread.c:1225: multiple definition of `pthread_attr_destroy'
../mysys/libmysys.a(my_winthread.c.obj):/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:47: first defined here
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_attr_getstacksize':
/home/uwe/mxe-git/tmp-gcc-i686-w64-mingw32.static.posix/gcc-4.9.3/mingw-w64-v4.0.6/mingw-w64-libraries/winpthreads/src/thread.c:1289: multiple definition of `pthread_attr_getstacksize'
../mysys/libmysys.a(my_winthread.c.obj):/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:41: first defined here
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_attr_setstacksize':
/home/uwe/mxe-git/tmp-gcc-i686-w64-mingw32.static.posix/gcc-4.9.3/mingw-w64-v4.0.6/mingw-w64-libraries/winpthreads/src/thread.c:1296: multiple definition of `pthread_attr_setstacksize'
../mysys/libmysys.a(my_winthread.c.obj):/home/uwemake[4]: Leaving directory '/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src.build'
/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:34: first defined here
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_create':
/home/uwe/mxe-git/tmp-gcc-i686-w64-mingw32.static.posix/gcc-4.9.3/mingw-w64-v4.0.6/mingw-w64-libraries/winpthreads/src/thread.c:1426: multiple definition of `pthread_create'
../mysys/libmysys.a(my_winthread.c.obj):/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:140: first defined here
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_join':
/home/uwe/mxe-git/tmp-gcc-i686-w64-mingw32.static.posix/gcc-4.9.3/mingw-w64-v4.0.6/mingw-w64-libraries/winpthreads/src/thread.c:1533: multiple definition of `pthread_join'
../mysys/libmysys.a(my_winthread.c.obj):/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:237: first defined here
collect2: error: ld returned 1 exit status
libmysql/CMakeFiles/libmysql_api_test.dir/build.make:113: recipe for target 'libmysql/libmysql_api_test.exe' failed
make[4]: *** [libmysql/libmysql_api_test.exe] Error 1
[ 98%] Linking CXX executable my_print_defaults.exe
make[4]: Leaving directory '/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src.build'
CMakeFiles/Makefile2:747: recipe for target 'libmysql/CMakeFiles/libmysql_api_test.dir/all' failed
make[3]: *** [libmysql/CMakeFiles/libmysql_api_test.dir/all] Error 2
make[3]: *** Waiting for unfinished jobs....
cd /home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src.build/extra && /home/uwe/mxe-git/usr/x86_64-unknown-linux-gnu/bin/cmake -E cmake_link_script CMakeFiles/my_print_defaults.dir/link.txt --verbose=1
[ 98%] Built target mysqlclient
/home/uwe/mxe-git/usr/x86_64-unknown-linux-gnu/bin/cmake -E remove -f CMakeFiles/my_print_defaults.dir/objects.a
/home/uwe/mxe-git/usr/bin/i686-w64-mingw32.static.posix-ar cr CMakeFiles/my_print_defaults.dir/objects.a @CMakeFiles/my_print_defaults.dir/objects1.rsp
/home/uwe/mxe-git/usr/bin/i686-w64-mingw32.static.posix-g++   -Wall -Wextra -Wformat-security -Wvla -Woverloaded-virtual -Wno-unused-parameter -O2 -g -DNDEBUG -DDBUG_OFF   -Wl,--whole-archive CMakeFiles/my_print_defaults.dir/objects.a -Wl,--no-whole-archive  -o my_print_defaults.exe -Wl,--out-implib,libmy_print_defaults.dll.a -Wl,--major-image-version,0,--minor-image-version,0 @CMakeFiles/my_print_defaults.dir/linklibs.rsp
[ 98%] Linking CXX executable perror.exe
cd /home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src.build/extra && /home/uwe/mxe-git/usr/x86_64-unknown-linux-gnu/bin/cmake -E cmake_link_script CMakeFiles/perror.dir/link.txt --verbose=1
/home/uwe/mxe-git/usr/x86_64-unknown-linux-gnu/bin/cmake -E remove -f CMakeFiles/perror.dir/objects.a
/home/uwe/mxe-git/usr/bin/i686-w64-mingw32.static.posix-ar cr CMakeFiles/perror.dir/objects.a @CMakeFiles/perror.dir/objects1.rsp
/home/uwe/mxe-git/usr/bin/i686-w64-mingw32.static.posix-g++   -Wall -Wextra -Wformat-security -Wvla -Woverloaded-virtual -Wno-unused-parameter -O2 -g -DNDEBUG -DDBUG_OFF   -Wl,--whole-archive CMakeFiles/perror.dir/objects.a -Wl,--no-whole-archive  -o perror.exe -Wl,--out-implib,libperror.dll.a -Wl,--major-image-version,0,--minor-image-version,0 @CMakeFiles/perror.dir/linklibs.rsp
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_exit':
../mysys/libmysys.a(my_winthread.c.obj):/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:185: first defined here
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_cancel':
/home/uwe/mxe-git/tmp-gcc-i686-w64-mingw32.static.posix/gcc-4.9.3/mingw-w64-v4.0.6/mingw-w64-libraries/winpthreads/src/thread.c:1078: multiple definition of `pthread_cancel'
../mysys/libmysys.a(my_winthread.c.obj):/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:247: first defined here
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_attr_init':
/home/uwe/mxe-git/tmp-gcc-i686-w64-mingw32.static.posix/gcc-4.9.3/mingw-w64-v4.0.6/mingw-w64-libraries/winpthreads/src/thread.c:1215: multiple definition of `pthread_attr_init'
../mysys/libmysys.a(my_winthread.c.obj):/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:25: first defined here
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_attr_destroy':
/home/uwe/mxe-git/tmp-gcc-i686-w64-mingw32.static.posix/gcc-4.9.3/mingw-w64-v4.0.6/mingw-w64-libraries/winpthreads/src/thread.c:1225: multiple definition of `pthread_attr_destroy'
../mysys/libmysys.a(my_winthread.c.obj):/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:47: first defined here
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_attr_getstacksize':
/home/uwe/mxe-git/tmp-gcc-i686-w64-mingw32.static.posix/gcc-4.9.3/mingw-w64-v4.0.6/mingw-w64-libraries/winpthreads/src/thread.c:1289: multiple definition of `pthread_attr_getstacksize'
../mysys/libmysys.a(my_winthread.c.obj):/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:41: first defined here
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_attr_setstacksize':
/home/uwe/mxe-git/tmp-gcc-i686-w64-mingw32.static.posix/gcc-4.9.3/mingw-w64-v4.0.6/mingw-w64-libraries/winpthreads/src/thread.c:1296: multiple definition of `pthread_attr_setstacksize'
../mysys/libmysys.a(my_winthread.c.obj):/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:34: first defined here
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_create':
/home/uwe/mxe-git/tmp-gcc-i686-w64-mingw32.static.posix/gcc-4.9.3/mingw-w64-v4.0.6/mingw-w64-libraries/winpthreads/src/thread.c:1426: multiple definition of `pthread_create'
../mysys/libmysys.a(my_winthread.c.obj):/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:140: first defined here
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_join':
/home/uwe/mxe-git/tmp-gcc-i686-w64-mingw32.static.posix/gcc-4.9.3/mingw-w64-v4.0.6/mingw-w64-libraries/winpthreads/src/thread.c:1533: multiple definition of `pthread_join'
../mysys/libmysys.a(my_winthread.c.obj):/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:237: first defined here
collect2: error: ld returned 1 exit status
extra/CMakeFiles/my_print_defaults.dir/build.make:107: recipe for target 'extra/my_print_defaults.exe' failed
make[4]: *** [extra/my_print_defaults.exe] Error 1
make[4]: Leaving directory '/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src.build'
CMakeFiles/Makefile2:909: recipe for target 'extra/CMakeFiles/my_print_defaults.dir/all' failed
make[3]: *** [extra/CMakeFiles/my_print_defaults.dir/all] Error 2
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_exit':
/home/uwe/mxe-git/tmp-gcc-i686-w64-mingw32.static.posix/gcc-4.9.3/mingw-w64-v4.0.6/mingw-w64-libraries/winpthreads/src/thread.c:961: multiple definition of `pthread_exit'
../mysys/libmysys.a(my_winthread.c.obj):/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:185: first defined here
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_cancel':
/home/uwe/mxe-git/tmp-gcc-i686-w64-mingw32.static.posix/gcc-4.9.3/mingw-w64-v4.0.6/mingw-w64-libraries/winpthreads/src/thread.c:1078: multiple definition of `pthread_cancel'
../mysys/libmysys.a(my_winthread.c.obj):/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:247: first defined here
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_attr_init':
/home/uwe/mxe-git/tmp-gcc-i686-w64-mingw32.static.posix/gcc-4.9.3/mingw-w64-v4.0.6/mingw-w64-libraries/winpthreads/src/thread.c:1215: multiple definition of `pthread_attr_init'
../mysys/libmysys.a(my_winthread.c.obj):/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:25: first defined here
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_attr_destroy':
/home/uwe/mxe-git/tmp-gcc-i686-w64-mingw32.static.posix/gcc-4.9.3/mingw-w64-v4.0.6/mingw-w64-libraries/winpthreads/src/thread.c:1225: multiple definition of `pthread_attr_destroy'
../mysys/libmysys.a(my_winthread.c.obj):/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:47: first defined here
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_attr_getstacksize':
/home/uwe/mxe-git/tmp-gcc-i686-w64-mingw32.static.posix/gcc-4.9.3/mingw-w64-v4.0.6/mingw-w64-libraries/winpthreads/src/thread.c:1289: multiple definition of `pthread_attr_getstacksize'
../mysys/libmysys.a(my_winthread.c.obj):/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:41: first defined here
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_attr_setstacksize':
/home/uwe/mxe-git/tmp-gcc-i686-w64-mingw32.static.posix/gcc-4.9.3/mingw-w64-v4.0.6/mingw-w64-libraries/winpthreads/src/thread.c:1296: multiple definition of `pthread_attr_setstacksize'
../mysys/libmysys.a(my_winthread.c.obj):/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:34: first defined here
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_create':
/home/uwe/mxe-git/tmp-gcc-i686-w64-mingw32.static.posix/gcc-4.9.3/mingw-w64-v4.0.6/mingw-w64-libraries/winpthreads/src/thread.c:1426: multiple definition of `pthread_create'
../mysys/libmysys.a(my_winthread.c.obj):/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:140: first defined here
/home/uwe/mxe-git/usr/lib/gcc/i686-w64-mingw32.static.posix/4.9.3/../../../../i686-w64-mingw32.static.posix/lib/../lib/libpthread.a(libwinpthread_la-thread.o): In function `pthread_join':
/home/uwe/mxe-git/tmp-gcc-i686-w64-mingw32.static.posix/gcc-4.9.3/mingw-w64-v4.0.6/mingw-w64-libraries/winpthreads/src/thread.c:1533: multiple definition of `pthread_join'
../mysys/libmysys.a(my_winthread.c.obj):/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src/mysys/my_winthread.c:237: first defined here
collect2: error: ld returned 1 exit status
extra/CMakeFiles/perror.dir/build.make:107: recipe for target 'extra/perror.exe' failed
make[4]: *** [extra/perror.exe] Error 1
make[4]: Leaving directory '/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src.build'
CMakeFiles/Makefile2:868: recipe for target 'extra/CMakeFiles/perror.dir/all' failed
make[3]: *** [extra/CMakeFiles/perror.dir/all] Error 2
make[3]: Leaving directory '/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src.build'
Makefile:149: recipe for target 'all' failed
make[2]: *** [all] Error 2
make[2]: Leaving directory '/home/uwe/mxe-git/tmp-libmysqlclient-i686-w64-mingw32.static.posix/mysql-connector-c-6.1.6-src.build'
Makefile:548: recipe for target 'build-only-libmysqlclient_i686-w64-mingw32.static.posix' failed
make[1]: *** [build-only-libmysqlclient_i686-w64-mingw32.static.posix] Error 2

```

**Comment by Majestio** [comment]
The patch (https://github.com/mxe/mxe/pull/1009) still does not allow to build :( Who ever it doing now?

**Comment by tonytheodore** [comment]
@Majestio see #1488.

**Comment by tonytheodore** [comment]
fixed in #1488

**Comment by Majestio** [comment]
tonytheodore, thx!
